cmd/git-sync/main.go

```diff
@@ -621,6 +630,22 @@ func main() {
 		}
 
 		if initialSync {
+			// Wait for hooks to complete at least once, if not nil, before checking whether to stop program
+			// Assumes that if hook channels are not nil, they will have at least one value before getting closed
+			if exechookChannel != nil {
+				exechookChannelFinishedSuccessfully := <-exechookChannel
+				if !exechookChannelFinishedSuccessfully {
```

Comment:
nit: "webhook" "exechook"

Reply:
I do not follow what this comment means; I suspect you want exechookChannel != nil to be exechook != nil. May you please clarify?

Reply:
I meant in the log messages - "web hook" -> "webhook"